answer/166ryoryo/JS/03/toBin.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+<meta charset="utf-8">
+<title></title>
+<meta name="keywords" content="">
+<meta name="description" content="">
+</head>
+<body>
+  <form name="math">
+    <input type="text" name="" value="0">：10進数<br>
+    <input type="text" name="" value="">：2進数<br>
+    <input type="button" value="変換" onclick="kazu()"><br>
+  </form>
+  <script>
+    function kazu(){
```

Comment:
```suggestion
    function kazu() {
```
function 関数名() {}
　　　　　　　  ↑
関数を書くときはここにスペースが必要です